README.md

```diff
@@ -1,2 +1,293 @@
-# bedrock-kms-http
-HTTP APIs for Bedrock Key Management
+# Bedrock Key Management System API module _(bedrock-kms-http)_
+> HTTP APIs for Bedrock Key Management.
+
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [Commercial Support](#commercial-support)
+- [License](#license)
+
+## Install
+
+- Node.js 12+ is required.
+
+### NPM
+
+To install via NPM:
+
+```
+npm install --save bedrock-kms-http
+```
+
+### Development
+
+To install locally (for development):
+
+```
+git clone https://github.com/digitalbazaar/bedrock-kms-http.git
+cd bedrock-kms-http
+npm install
+```
+
+## Usage
+
+In `lib/index.js` (or `main.js`, as appropriate):
+
+```js
+require('bedrock-kms-http');
```

Comment:
I think we should recommend ESM everywhere; CommonJS should be considered deprecated.

```suggestion
import 'bedrock-kms-http';
```